SU-HSDO/stanford_mrc
# MRCD8-200 News Fixes

# READY FOR REVIEW

# Summary
- Hard-coded height and width of news node image since there is no image style being applied to ensure that it always stays inside of the Decanter container
- Instead of using Flexbox mixin to drop to a block display, keeping flex positioning for this node type since the "|" character is now added to the template

# Needed By (Date)
- End of sprint (1/17)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Test a sample news node with a large image
3. Ensure that the image does not extend outside of the container and that the date and source remain inline with one another unless too long (then dynamically drops)

# Affected Projects or Products
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-200

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
